### DISCUSSION TITLE
definePageMeta is throwing eslint error message "Cannot find name definePageMeta"

### DISCUSSION THREAD

**Discussion by ramyohan90** [nuxt/nuxt#24975] [General]
I am using Nuxt@3.9.0 & Vue@3.4.0

I have a vue component under pages (pages/login.vue). I have enabled the Nuxt Extensions in VS Code to support intellisense in code space.

I need to call a middleware in the vue component and hence used definePageMeta to call the middleware. 

Although I can able to call the middleware, VS Code is throwing error message "Cannot find name definePageMeta".

Please let know what kind of setup I should do, so that VS Code resolves this error.

As per Nuxt Documentation, we can call middleware in the vue components under pages and the router will identify definePageMeta. I followed the same process, but VS Code shows eslint error. 

Please help.

![nuxt-defineMeta](https://github.com/nuxt/nuxt/assets/32775670/d03586d8-13d9-44ff-8031-a9ea89f6befd)

**Comment by ramyohan90** [comment]
Closing this. Nuxt dev tools was not installed properly in VS Code. Hence intellisense failed.

**Reply by nbro** [reply]
@ramyohan90 Which extension have you exactly installed? I installed [the official nuxt VS Code extension](https://marketplace.visualstudio.com/items?itemName=Nuxtr.nuxtr-vscode), but VS Code still complains about it.

**Reply by aryaadinulfadlan** [reply]
@ramyohan90 which nuxt extension did you use ?

**Reply by gregmsanderson** [reply]
For anyone else getting this, simply _restarting_ VSCode can fix it. It did for me. Sigh.

**Reply by Niatomi** [reply]
It actually not some exact extension, but rather some vue/nuxt related extensions [like there](https://stackoverflow.com/questions/76569729/nuxt-3-useroute-import-cannot-find). Mine was [Vue - Official](https://marketplace.visualstudio.com/items?itemName=Vue.volar) so I just simply reinstalled it and no more unkown names
